shared/src/main/scala/squants/unitgroups/ImplicitDimensions.scala

```diff
@@ -0,0 +1,104 @@
+package squants.unitgroups
+
+import squants.Dimension
+import squants.electro._
+import squants.energy._
+import squants.information.{DataRate, Information}
+import squants.mass.{AreaDensity, ChemicalAmount, Density, Mass}
+import squants.motion._
+import squants.photo._
+import squants.radio._
+import squants.space._
+import squants.thermal.{Temperature, ThermalCapacity}
+import squants.time.{Frequency, Time}
+
+/**
+  * Puts [[Dimension]] objects into implicit scope
+  * so that implicit [[UnitGroup]] machinery works.
+  */
+object ImplicitDimensions {
```

Comment:
Just a general were question. If we were going to add a new `Quantity` would there be compilation errors if the implicit `Dimension` isn't defined here? I hope that's the case 😄